Source/React.cpp

```diff
@@ -170,6 +182,20 @@ PeleC::react_state(
         auto const& mask = dummyMask.array(mfi);
         auto const& fc = fctCount.array(mfi);
 
+        if (use_chem_mask) {
+          amrex::ParallelFor(
+            bx, [=] AMREX_GPU_DEVICE(int i, int j, int k) noexcept {
+              amrex::XDim3 point;
```

Comment:
can simplify: `const amrex::IntVect iv = {AMREX_D_DECL(plo[0] + (i + 0.5) * dx[0], etc)};` see other examples in the code.

Reply:
This should probably just happen once? No need to do it every step? if so, can we move to the init somewhere?

Reply:
I think @marchdf means `const amrex::RealVect = ...`. The AMREX_D_DECL is needed so the code also works in 2d. You can also use the pc_cmp_loc function which wraps this.